--- origsrc/xfconf-4.5.92/configure.ac	2008-11-09 17:08:01.000000000 -0600
+++ src/xfconf-4.5.92/configure.ac	2008-12-25 04:35:21.771750000 -0600
@@ -108,6 +108,13 @@
 AC_SUBST([PERL_EXTUTILS_DEPENDS_REQUIRED_VERSION])
 AC_SUBST([PERL_EXTUTILS_PKGCONFIG_REQUIRED_VERSION])
 
+# Options to pass when configuring Perl bindings
+AC_ARG_WITH([perl-options],
+    [AC_HELP_STRING([--with-perl-options=OPTIONS],
+                    [options to pass on command-line when generating Perl bindings build file])])
+PERL_MAKE_OPTIONS=$with_perl_options
+AC_SUBST(PERL_MAKE_OPTIONS)
+
 dnl check alignment for struct returns
 AC_DEFUN([BT_GLIB_CHECK_ALIGNOF],
 [
--- origsrc/xfconf-4.5.92/xfconf-perl/Makefile.am	2008-12-25 04:36:07.240500000 -0600
+++ src/xfconf-4.5.92/xfconf-perl/Makefile.am	2008-12-25 04:35:46.178000000 -0600
@@ -20,7 +20,7 @@
 	        test -f $$f || $(LN_S) $(srcdir)/$$f $$f; \
 	    done; \
 	fi
-	top_builddir="$(top_srcdir)" $(PERL) Makefile.PL
+	top_builddir="$(top_srcdir)" $(PERL) Makefile.PL $(PERL_MAKE_OPTIONS)
 
 all-local: Makefile.perl
 	$(MAKE) -f Makefile.perl
