--- origsrc/xfconf-4.5.91/xfconf-query/Makefile.am	2008-04-09 15:21:08.000000000 -0500
+++ src/xfconf-4.5.91/xfconf-query/Makefile.am	2008-10-31 14:27:13.641546900 -0500
@@ -13,8 +13,8 @@
 	-DLOCALEDIR=\"$(localedir)\"
 
 xfconf_query_LDADD = \
-	$(top_builddir)/xfconf/libxfconf-0.la \
 	$(top_builddir)/common/libxfconf-gvaluefuncs.la \
+	$(top_builddir)/xfconf/libxfconf-0.la \
 	$(GLIB_LIBS) \
 	$(DBUS_GLIB_LIBS)
 
